pkg/services/sqlstore/quota.go

```diff
@@ -156,6 +166,10 @@ func (ss *SQLStore) GetUserQuotaByTarget(ctx context.Context, query *models.GetU
 		if query.Target != alertRuleTarget || query.UnifiedAlertingEnabled {
 			// get quota used.
 			rawSQL := fmt.Sprintf("SELECT COUNT(*) as count from %s where user_id=?", dialect.Quote(query.Target))
+			// removing service accounts from the count
+			if query.Target == dialect.Quote("user") {
```

Comment:
We don't have to make changes here. User quotas only exists on org and global level

Reply:
I thought though user quota as far as i can tell is calling to get users? 🤔

Reply:
from this point for example? https://github.com/grafana/grafana/blob/main/pkg/services/quota/quotaimpl/quota.go#L117

Reply:
agree with @eleijonmarck that we need this: it's used for getting the number of users (on org and global level),